patches/server/0338-Optimize-Hoppers.patch

```diff
@@ -114,11 +114,11 @@ index a507d7f65a94e49ecd18cd18797b156474558390..a7ac6b528aecae528a17af157f8ec293
 +            if (!item.isEmpty()) {
 +                foundItem = true;
 +                ItemStack origItemStack = item;
-+                ItemStack itemstack = origItemStack;
++                ItemStack itemstack = origItemStack.copy();
```

Comment:
the issue says about use clone.. but NMS have a method for this.. my issue if can be this method or can be `cloneItemStack(true)`

Reply:
I just said `clone` because thats what bukkit uses but `copy` should work too